src/VirtualClient/VirtualClient.Monitors.UnitTests/Examples/nvidia-smi/query-c2c.txt

```diff
@@ -0,0 +1,11 @@
+GPU 0: NVIDIA GH200 480GB (UUID: GPU-8d852f73-f973-0769-9c78-5e52c7b36f16)
```

Comment:
what happens if we have multiple gpu, does parser handle that

Reply:
yes the logic is covered in the parser code - it will be parsed line and line and when gpu information is read, it updates here -
if (gpuMatch.Success)
{
    gpuNumber = int.Parse(gpuMatch.Groups["GPU"].Value.Trim());
    gpuName = gpuMatch.Groups["Name"].Value.Trim();
    gpuUuid = gpuMatch.Groups["UUID"].Value.Trim();
    continue;
} and in the metrics table we are adding c2c link like this "GPU {gpuNumber}: C2C Link {linkNumber} Speed" -- ensuring that GPU number and link number both are there.